packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -132,6 +135,7 @@ class InteractiveViewer extends StatefulWidget {
        assert(builder != null),
        assert(minScale != null),
        assert(minScale > 0),
+       assert(scrollFriction > 0),
```

Comment:
```suggestion
       assert(scrollFrictionCoefficient > 0),
```

Adding this suggestion in case you agree with https://github.com/flutter/flutter/pull/109443/files#r944361349, in this case you can use GitHub to batch the suggestions and automatically make the change.